### PR TITLE
Revert nextjs static page revert and apply fix

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Closes https://github.com/jbx-protocol/juice-interface/issues/1634

Was discussed [on discord](https://discord.com/channels/939317843059679252/1004091848366759977)

During my PR to get SEO working, I moved all the stuff out of _app.pages.tsx into each respective page as LanguageProvider blocks all children from pre-rendering. 

_This was a necessity to ensure that at least the SEO per page was rendered accordingly_

```
Aside:
Lingui i18n provider basically returns null if the default locale is not provider, which happens on the pre-render. This is easily remedied, but since our app is not ready for proper html static page rendering, it is not what we want (whole pre render blows up)
```

This meant that on each page switch, the onboard was soft destroyed (incorectly) and then it was getting called again which was causing it to blow up. Moving the NetworkProvider back to _app (but leaving everything else) solves this particular problem in time for now while we work toward solving other stuff.

I believe that the reason this solves the issue is that the provider is living between each page load (as expected) 

**TLDR:** This is fixed by applying the NetworkProvider at the most top level of the next app, rather than embedded in each page. _Reasons for initially embedding in each page can be seen in above ^_

### Before

https://user-images.githubusercontent.com/104132113/182831240-58e25949-fb84-42cf-afc1-abf13b0a165d.mov

### After

https://user-images.githubusercontent.com/104132113/182831514-4c6998a0-c022-402b-a371-44627978fd38.mov

### Proof of deploy

https://user-images.githubusercontent.com/104132113/182832964-6090d4cf-3d64-4f25-8e28-6c8df24bf411.mov

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
I've also tested deploy and that is all good again too! Feel free to use the rink deploy to test too

**Comment by wraeth-eth** [comment]
https://juice-interface-rinkeby-hw09ymnxr-peel.vercel.app/@wraeth

^ if things are correct, link unfurling should be working too, but unsure if github does it. I tried it in a private discord

**Comment by tomquirk** [comment]
Rebasing!

**Comment by tomquirk** [comment]
@wraeth-eth thanks for a really really great PR description. Super helpful. All seems to be working for me on first pass but lets keep an eye on things over the next day or 2 🙏 

Enabling auto-merge ✅✅✅✅✅✅

**Review by tomquirk — APPROVED** [review]
